keylime/cloud_verifier_tornado.py

```diff
@@ -998,6 +998,76 @@ def data_received(self, chunk: Any) -> None:
         raise NotImplementedError()
 
 
+async def update_agent_api_version(agent: Dict[str, Any], timeout: float = 60.0) -> (Dict[str, Any] | None):
+    agent_id = agent["agent_id"]
+
+    logger.warning("Agent %s API version bump detected, trying to update stored API version", agent_id)
+    kwargs = {}
+    if agent["ssl_context"]:
+        kwargs["context"] = agent["ssl_context"]
```

Comment:
`kwargs = {"context": agent["ssl_context"] if agent["ssl_context"]}`, but I am not sure that `"ssl_context" in agent` is always true ...

Reply:
`ssl_context` indeed can be `None` if `agent_mtls_cer` is disabled